packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## `8.10.2`
```

Comment:
Please use "Recent Changes" for the header. When this PR is merged our automation will update it.
```suggestion
## Recent Changes
```